plugins/swagger-apibody/src/util/ast.ts

```diff
@@ -0,0 +1,196 @@
+import { visit } from "recast";
+import { ASTNode, namedTypes, builders } from "ast-types";
+import * as K from "ast-types/gen/kinds";
+import { NodePath } from "ast-types/lib/node-path";
+
+const CONSTRUCTOR_NAME = "constructor";
+
+export function transformTemplateLiteralToStringLiteral(
+  templateLiteral: namedTypes.TemplateLiteral
+): namedTypes.StringLiteral {
+  const value = templateLiteral.quasis
+    .map((quasie, i) => {
+      const expression = templateLiteral.expressions[
+        i
```

Comment:
There is already a library called code-gen-utils
https://www.npmjs.com/package/@amplication/code-gen-utils

if these functions are missing, please update the library

Reply:
do you mean visit from recast ? cause it is part of the library not [@amplication/code-gen-utils]